plenum/server/node.py

```diff
@@ -2670,7 +2670,23 @@ def processOrdered(self, ordered: Ordered):
                              ordered.stateRootHash,
                              ordered.txnRootHash))
 
-        self.executeBatch(ThreePcBatch.from_ordered(ordered),
+        three_pc_batch = ThreePcBatch.from_ordered(ordered)
+        if self.db_manager.ledgers[AUDIT_LEDGER_ID].uncommittedRootHash is None:
```

Comment:
Could it happen so that `uncommittedRootHash` is not None but due to other batch in flight, so that we still need to manually apply transactions?

Reply:
I think no, because here we always have apply-commit cycle, so that even if this is an Ordered msg came in between catchup rounds during view change, each Ordered will be re-applied and committed immediately, so that it always end up with empty uncommitted state if the audit ledger.